pipeline/extract.py

```diff
@@ -53,12 +53,12 @@ def flatten_and_organize_data(plant_dict: dict) -> dict:
     return new_plant_dict
 
 
-def fetch_plant_data(current_plant):
+def fetch_plant_data(current_plant, session):
     """
     Fetches data for a single plant.
     """
     try:
-        response = requests.get(f"{API_URL}{current_plant}", timeout=20)
+        response = session.get(f"{API_URL}{current_plant}", timeout=20)
```

Comment:
Nice use of adding session to the extract script. Really speeds us request response times.